### DISCUSSION TITLE
Client to client communication using open vpn on android OS

### DISCUSSION THREAD

**Discussion by saad3344555** [schwabe/ics-openvpn#1663] [General]
I would like to enquire about creating server configuration in android device using your opensource repo. Following are some questions regarding the possibility of the this use case.

1. Is it possible to integrate server configuration file and establish a server on android device on which different clients will be connected?

2. Can we create a socket on vpn connections to listen to the data receive from another client app?

3. Can we communicate between the clients that are connected to the same vpn by means of sockets or anything?

If yes then please explain with any helpful material. Note we cant buy vpn server to achieve this. We need vpn configurations.

We are using ics-openvpn for android

**Comment by schwabe** [comment]
Duplicate of #1662
